run_wml_tests

```diff
@@ -155,70 +182,71 @@ class WesnothRunner:
                 if test.status != UnitTestResult.PASS:
                     raise NotImplementedError("run_tests doesn't yet support batching tests with non-zero statuses")
         expected_result = test_list[0].status
+
+        if expected_result == UnitTestResult.TIMEOUT and self.timeout == 0:
+            test_summary.skip_test(test_list)
+            if self.verbose >= Verbosity.NAMES_OF_TESTS_RUN:
+                print('Skipping test', test_list[0].name, 'because timeout is disabled')
+            return
+        if expected_result == UnitTestResult.FAIL_BROKE_STRICT and not options.strict_mode:
+            test_summary.skip_test(test_list)
+            if self.verbose >= Verbosity.NAMES_OF_TESTS_RUN:
+                print('Skipping test', test_list[0].name, 'because strict mode is disabled')
+            return
+
         args = self.common_args.copy()
         for test in test_list:
             args.append("-u")
             args.append(test.name)
+
         if self.timeout == 0:
-            if test.status == UnitTestResult.TIMEOUT:
-                test_summary.skip_test()
-                print('Skipping test', test_list[0].name, 'because timeout is disabled')
-                return
             timeout = None
+        elif len(test_list) == 1:
+            timeout = self.timeout
         else:
-            if test.status == UnitTestResult.FAIL_BROKE_STRICT and not options.strict_mode:
-                test_summary.skip_test()
-                print('Skipping test', test_list[0].name, 'because strict mode is disabled')
-                return
-            if len(test_list) == 1:
-                timeout = self.timeout
-            else:
-                timeout = self.batch_timeout
-        if self.verbose > 0:
+            timeout = self.batch_timeout
+
+        if self.verbose >= Verbosity.NAMES_OF_TESTS_RUN:
             if len(test_list) == 1:
                 print("Running test", test_list[0].name)
             else:
                 print("Running {count} tests ({names})".format(count=len(test_list),
                     names=", ".join([test.name for test in test_list])))
-            if self.verbose > 1:
-                print(repr(args))
+        if self.verbose >= Verbosity.SCRIPT_DEBUGGING:
+            print(repr(args))
```

Comment:
`args` is a list, right? So I think `print(*args)` or `print(' '.join(args))` would be better.

Reply:
`repr` prints out whitespace in a different way to the other two, and when debugging the script I'd rather have `repr` than miss this edge case:
```
>>> x = ["a", "b c", "d"]
>>> print (*x)
a b c d
>>> print (repr(x))
['a', 'b c', 'd']
```

Reply:
Then just replace `args` with `[repr(x) for x in args]` in either of the above print calls? Just not having the brackets and commas would help, I think.

Reply:
What is the use-case that needs help here? Who is expected to be using this level of verbosity?

Reply:
Probably only someone who's debugging the tool itself?

Reply:
My opinion is unchanged then - I think `print(repr(args))` gives the output most useful for that target audience.